apps/src/aichat/views/presentation/presentation-view.module.scss

```diff
@@ -1,9 +1,13 @@
 @import 'color';
 
 .modelCardTitle {
-    background-color: $neutral_light;
```

Comment:
Would you mind confirming that @markabarrett that he wants a white background for the model card title in presentation view? I don't see title included the figma mockups (I may be missing this). Thanks!

Reply:
Thanks for catching this! Going to merge this is as-is for now and follow-up with Mark on further tweaks once he's back.